tests/test_fields.py

```diff
@@ -47,6 +47,17 @@ def test_include_resource_linkage_single(self, post):
         assert result['data']
         assert result['data']['id'] == str(post.author.id)
 
+    def test_include_resource_linkage_single_with_schema(self, post):
+        field = Relationship(
+            related_url='/posts/{post_id}/author/',
+            related_url_kwargs={'post_id': '<id>'},
+            include_resource_linkage=True, type_='people', schema='PostSchema'
```

Comment:
I do not believe it should, as the base Schema provides a get_attribute implementation which simply uses the utils.get_value function. Do you want me to add a test case with schema that overrides get_attribute?